fdb-record-layer-lucene/src/main/java/org/apache/lucene/codecs/lucene84/LuceneOptimizedPostingsReader.java

```diff
@@ -2016,6 +2016,7 @@ public long ramBytesUsed() {
 
     @Override
     public void checkIntegrity() throws IOException {
+        // Should this checksum verification be skipped for fdb?
```

Comment:
Based on the other places, I think that would make sense to align this with rest of our implementations for `checkIntegrity`